src/renderer/editor/components/inspectors/script-inspector.tsx

```diff
@@ -366,6 +380,28 @@ export class ScriptInspector<T extends (Scene | Node), S extends IScriptInspecto
         );
     }
 
+    /**
+     * Gets all nodes in the current scene, or returns an empty array if no scene
+     */
+    getSceneNodes(allowedType?: "TransformNode" | "Mesh" | "Light" | "Camera" | "AbstractMesh"): IInspectorListItem<string>[] {
```

Comment:
Please try to always provide the accessors like private/public etc.
Private methods begin with a "_": private _getSceneNodes(...) { ... }
Also please try to always put semicolons;
The project is missing some ESlint/prettier but this will added ASAP to help :)